src/containers/Network/BarChartVersion.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  Tooltip,
+  TooltipProps,
+  Label,
+  ResponsiveContainer,
+  Legend,
+  Cell,
+} from 'recharts'
+import {
+  ValueType,
+  NameType,
+} from 'recharts/src/component/DefaultTooltipContent'
+import './css/barchart.scss'
+
+interface Props {
+  data: any[]
+  stableVersion: string | null
+}
+
+interface BarCoordinates {
+  x: number
+  y: number
+}
+
+const CustomTooltip = ({
+  active,
+  payload,
+  label,
+}: TooltipProps<ValueType, NameType>) => {
+  const { t } = useTranslation()
+  if (active) {
+    return (
+      <div className="custom-tooltip">
+        <p className="label">{t('version_display', { version: label })}</p>
+        <p className="value">
+          {t('nodes_count', { count: payload?.[0].payload.count })}
+        </p>
+      </div>
+    )
+  }
+  return null
+}
+
+const renderLegend = (stableVersion: string | null, t: any) => (
+  <div className="legend">
+    <div className="legend-text">
+      <span>{t('current_stable_version')}:</span>
+      <span style={{ color: '#19FF83' }}>
```

Comment:
This color should use the constants defined below - might be worth bringing their definitions up to a higher level to accommodate that.